### ISSUE TITLE
Error when initializing modules

### ISSUE THREAD

**Issue by jvalleroy** [freedombox/Plinth#671]
I missed this issue when merging in a recent change, to only add services (for the firewall) for modules that have completed setup. Strangely I didn't see this issue at all when testing in vagrant.

When running Plinth out of the source folder, I get this error:
```
Traceback (most recent call last):
  File "./run", line 21, in <module>
    plinth.__main__.main()
  File "/home/jvalleroy/fbx/plinth/plinth/__main__.py", line 323, in main
    module_loader.load_modules()
  File "/home/jvalleroy/fbx/plinth/plinth/module_loader.py", line 77, in load_modules
    _initialize_module(module_name, modules[module_name])
  File "/home/jvalleroy/fbx/plinth/plinth/module_loader.py", line 135, in _initialize_module
    init()
  File "/home/jvalleroy/fbx/plinth/plinth/modules/privoxy/__init__.py", line 78, in init
    if service.is_enabled():
AttributeError: 'NoneType' object has no attribute 'is_enabled'
```

It's due to these lines in the init() for privoxy (and other modules):
```
def init():
    ...
    if service.is_enabled():
        add_shortcut()
```

**Comment by 500swapnil** [comment]
Hi I would like to help with this issue. Can u tell me how to proceed?

**Comment by jvalleroy** [comment]
@500swapnil You will need to look at each module that was modified in this commit: https://github.com/freedombox/Plinth/commit/941259df1ebf17fb8869e910f1ac66e8847ecbc1
Each module has a file ```plinth/modules/<module_name>/__init__.py```, which contains an ```init()``` function.
See which ones have the ```add_shortcut()``` line at the end of ```init()```.

For each of those, we just need to add an indent to the 2 lines, so they are under the ```if setup_helper.get_state() != 'needs-setup'``` condition.

It will be easy to test to confirm the issue/fix if you have vagrant set up. For example:
```
vagrant@freedombox:/vagrant$ sudo ./run --debug
...
AttributeError: 'NoneType' object has no attribute 'is_enabled'
```
Due to this issue, plinth will not start when run with the ```--debug``` flag.

**Comment by 500swapnil** [comment]
#672

**Comment by jvalleroy** [comment]
Fixed in #672.
